bridge_adaptivity/module/engines/engine_vpal.py

```diff
@@ -0,0 +1,138 @@
+import logging
+import requests
+import urlparse
+
+from module.engines.interface import EngineInterface
+
+log = logging.getLogger(__name__)
+
+
+ACTIVITY_PARAMETERS = (
+    'collection',
+    'name',
+    'tags',
+    'type',
+    'difficulty',
+)
+
+SEQUENCE_ITEM_PARAMS = (
+    'learner',
+    'activity',
+    'score',
+)
+
+TYPES = {
+    'A': 'pre-assessment',
+    'Z': 'post-assessment'
+}
+
+
+class EngineVPAL(EngineInterface):
+    """
+    VPAL adaptive engine.
+
+    To use VPAL engine with the Bridge for Adaptivity configure following parameters in the secure settings::
+
+        ENGINE_MODULE = 'module.engines.engine_vpal'
+
+        # ENGINE_DRIVER is a string with the name of driver class in the engine module
+        ENGINE_DRIVER = 'EngineVPAL'
+
+        # ENGINE_SETTINGS is a dict, with the initial params for driver initialization
+        ENGINE_SETTINGS = {
```

Comment:
Does it make sense to add the authentication token here too? (see authentication notes on endpoint document)